bigbluebutton-html5/imports/api/audio/client/bridge/service.js

```diff
@@ -24,6 +24,9 @@ const getCurrentAudioSessionNumber = () => sessionStorage.getItem(AUDIO_SESSION_
 const reloadAudioElement = (audioElement) => {
   if (audioElement && (audioElement.readyState > 0)) {
     audioElement.load();
+    if (audioElement.paused) {
+      audioElement.play();
```

Comment:
`play` throws. Catch the error to avoid spurious exceptions/crashes, and log it (e.g.: `ogCode: audio_reload_element_error`).